src/xdocs/config.xml

```diff
@@ -605,6 +619,18 @@
   ...
 &lt;/module&gt;
         </source>
+        <p>
+          To configure <code>TreeWalker</code> so that it
+          skips files with java parsing exceptions with severity level <code>ignore</code>:
```

Comment:
I think we need more examples and more verbage.

First example should show only `skipFileOnJavaParseException` on, why you would set it on (we should use verbage like new Java features we don't support yet), and what happens if it is not set on (maybe a 2nd example).

Next example should show how you can hide these exceptions if you don't want to see them as errors. This would set `javaParseExceptionSeverity`.

Reply:
done

Reply:
@rnveach please see the site and resolve this if you are good 
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/8a12016_2024131549/index.html
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/8a12016_2024131549/config.html#TreeWalker_Properties